aio/aio-proxy/aio_proxy/search/helpers.py

```diff
@@ -49,9 +41,11 @@ def sort_and_execute_search(search, offset: int, page_size: int) -> Tuple:
     search = search[offset : (offset + page_size)]
     results = search.execute()
     total_results = results.hits.total.value
-    response = [
-        hit.to_dict(skip_empty=False, include_meta=False) for hit in results.hits
-    ]
+    response = []
```

Comment:
La boucle for qui suit m'échappe un peu, peut etre un commentaire ou des variables plus claires ? Mais j'ai l'impression que ce sont les variables elastic, est ce bien ca ?

Reply:
Oui en effet, c'est en partie des variables Elastic !

Reply:
ok